parsely/src/test/java/com/parsely/parselyandroid/AndroidDeviceInfoRepositoryTest.kt

```diff
@@ -0,0 +1,78 @@
+package com.parsely.parselyandroid
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.robolectric.shadows.ShadowBuild
+
+private const val SDK_VERSION = 33
+private const val MANUFACTURER = "test manufacturer"
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [SDK_VERSION])
+internal class AndroidDeviceInfoRepositoryTest {
+
+    private lateinit var sut: AndroidDeviceInfoRepository
```

Comment:
Does this need to be a property or could we use a local value for it?

Reply:
That's totally OK, thanks for the changes!

I try and avoid the `var` keyword as much as possible, so anytime I see that in a PR review, I make it a point to check if we could remove it. I do something similar with properties, but I think sometimes they are the practical choice and commenting on those can get nitpicky, so I don't do it as much. Just thought I'd provide some context into how I review these.